opentelemetry-sdk/Cargo.toml

```diff
@@ -53,6 +53,7 @@ rt-tokio = ["tokio", "tokio-stream"]
 rt-tokio-current-thread = ["tokio", "tokio-stream"]
 rt-async-std = ["async-std"]
 internal-logs = ["tracing"]
+spec_unstable_metrics_views = ["metrics"]
```

Comment:
in a follow up, we need to document the general meaning of the feature flags. (Not in this PR)